src/main/java/frc/robot/RobotConstants.java

```diff
@@ -1,5 +1,9 @@
 package frc.robot;
 
 public class RobotConstants {
-    public static final int EXAMPLE_CONSTANT = 1;
+    //public static final int EXAMPLE_CONSTANT = 1;
+    public static final int MOTOR_CHANNEL_FL = 1;
+    public static final int MOTOR_CHANNEL_FR = 2;
+    public static final int MOTOR_INVERTED_FL = 3;
+    public static final int MOTOR_INVERTED_RR = 4;
```

Comment:
You're missing some stuff here. There are 4 drive motors total, so there should be 4 motor channels and 4 motor inversions.

Also the motor inversions should be booleans